quotes.json

```diff
@@ -537,6 +537,12 @@
     "author": "Arthur Schopenhauer",
     "text": "Every man takes the limits of his own field of vision for the limits of the world."
   },
+  {
+    "author": "Arthur Schopenhauer",
+    "source": "https://www.goodreads.com/author/quotes/11682.Arthur_Schopenhauer?page=2 ",
+    "tags": "pain, life",
+    "text": "Life swings like a pendulum backward and forward between pain and boredom."
+  },
```

Comment:
This is not a very inspiring quote ... More like something an Emo/Goth kid would say. 
[![image](https://user-images.githubusercontent.com/194400/109065032-106a7c80-76e3-11eb-8460-1bcc164efe37.png)](https://youtu.be/-3IuG0qw6GM)

```suggestion
```